code/modules/mob/living/carbon/human/human.dm

```diff
@@ -3,6 +3,7 @@
 	name = "unknown"
 	real_name = "unknown"
 	voice_name = "unknown"
+	var/fartCooldown = 20 SECONDS
```

Comment:
by convention, new vars typically go with other new vars not in the middle of the inherited vars

just scoot this down a few lines

Reply:
Moved it to line 13